### PR TITLE
added GTM code snippets, removed GA tracking code

### PR DESCRIPTION
**Pull request by Isterius**
- [ ] @albertpalka 
- [ ] @grodowski

### CODE REVIEW

File: source/partials/_m-google-tag-manager-body [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PBJW87D"
```
Comment (grodowski) [inline source/partials/_m-google-tag-manager-body]: Why you no exclude it in development?

Comment (Isterius) [inline source/partials/_m-google-tag-manager-body]: GA wasn't excluded so i didn't know, but your right, now it's excluded

Comment (Isterius) [inline source/partials/_m-google-tag-manager-body]: @grodowski is it ok now? ;)

Comment (grodowski) [inline source/partials/_m-google-tag-manager-body]: Looks good

**Pull request merged**